catalog/imaging/catalog-video-transformer/src/test/java/com/connexta/transformer/video/TestStanag4609ProcessorImpl.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.codice.ddf.libs.klv.KlvContext;
+import org.codice.ddf.libs.klv.KlvDecodingException;
+import org.codice.ddf.libs.klv.data.Klv;
+import org.codice.ddf.libs.klv.data.set.KlvLocalSet;
+import org.junit.Test;
+
+import com.connexta.alliance.libs.stanag4609.DecodedKLVMetadataPacket;
+
+public class TestStanag4609ProcessorImpl {
+
+    private static final Stanag4609Processor STANAG_4609_PROCESSOR = new Stanag4609ProcessorImpl();
+
+    @Test
+    public void testRegularHandlerForCallDataElementHandlers() throws KlvDecodingException {
+
+        KlvHandler klvHandler = mock(KlvHandler.class);
+
+        KlvHandler defaultKlvHandler = mock(KlvHandler.class);
+
+        String fieldName = "field";
+
+        STANAG_4609_PROCESSOR.callDataElementHandlers(Collections.singletonMap(fieldName,
+                klvHandler), defaultKlvHandler, KlvUtilities.createTestFloat(fieldName, 100D));
+
+        verify(klvHandler, atLeastOnce()).accept(any());
```

Comment:
For these tests, it might be a good idea to check that the `KlvDataElement` passed to the handler is what you expect.